client/settings/general-settings-section/index.js

```diff
@@ -3,95 +3,158 @@
  */
 import React, { useContext } from 'react';
 import styled from '@emotion/styled';
-import { __ } from '@wordpress/i18n';
-import { Card } from '@wordpress/components';
+import { Card, CheckboxControl } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
+import PaymentMethodFeesPill from 'wcstripe/components/payment-method-fees-pill';
 import CardBody from '../card-body';
-import CardsIcon from '../../payment-method-icons/cards';
-import UPEOptInBanner from '../upe-opt-in-banner';
+import UpeOptInBanner from '../upe-opt-in-banner';
 import UpeToggleContext from '../upe-toggle/context';
+import {
+	useEnabledPaymentMethods,
+	useGetAvailablePaymentMethods,
+} from './data-mock';
+import PaymentMethodDescription from './payment-method-description';
+import SectionHeading from './section-heading';
+import PaymentMethodsMap from '../../payment-methods-map';
+import PaymentMethodSetupHelp from './payment-method-setup-help';
 
-const GeneralSettingsSectionWrapper = styled.div`
-	display: flex;
-	flex-direction: column;
+const StyledUpeOptInBanner = styled( UpeOptInBanner )`
+	max-width: 100%;
+	margin: 0;
 `;
 
-const CardBodyWrapper = styled( CardBody )`
-	display: flex;
+const StyledCard = styled( Card )`
+	margin-bottom: 12px;
+`;
 
-	> * {
-		margin-bottom: 0px;
+const PaymentMethodsList = styled.ul`
+	margin: 0;
+
+	> li {
+		margin: 0;
+		padding: 16px 24px 14px 24px;
+
+		@media ( min-width: 660px ) {
+			padding: 24px 24px 24px 24px;
+		}
+
+		&:not( :last-child ) {
+			box-shadow: inset 0 -1px 0 #e8eaeb;
+		}
 	}
 `;
 
-const PaymentMethodText = styled.div`
-	flex: 0 0 100%;
+const PaymentMethodWrapper = styled.div`
+	display: flex;
+	flex-direction: column;
+	flex-wrap: nowrap;
 
 	@media ( min-width: 660px ) {
-		flex: 1 1 auto;
-		margin-left: 12px;
+		flex-direction: row;
+		align-items: center;
 	}
 `;
 
-const PaymentMethodLabel = styled.div`
-	color: $gray-900;
```

Comment:
This was not valid - fixed and moved the component to `client/settings/general-settings-section/payment-method-description.js`